flink-runtime/src/main/java/org/apache/flink/runtime/state/changelog/StateChangelogStorageLoader.java

```diff
@@ -99,4 +101,24 @@ public static StateChangelogStorage<?> load(
             return factory.createStorage(configuration, metricGroup);
         }
     }
+
+    @Nonnull
+    public static StateChangelogStorageView<?> load(ChangelogStateHandle changelogStateHandle)
```

Comment:
Can we use a different name to distinguish this method with above `load` method?

Reply:
replaced `load` with `loadFromStateHandle`